src/Ombi/ClientApp/src/app/discover/components/discover/discover.component.scss

```
@@ -1,40 +1,92 @@
+.skeleton-row {
+    display: flex;
+    gap: 12px;
+    overflow: hidden;
 }
 
+.skeleton-card {
+    flex: 0 0 180px;
+}
+
+// Hide PrimeNG carousel indicators globally on this page
+::ng-deep .p-carousel-indicators {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)